### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by ReemMohamedAbdelfatah**
Added popup menu for mobile and desktop versions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,127 +47,92 @@ <h1>I'm Kenneth Glad to see you!</h1>
...
+
+        <div id="sectionWrap" class="modal">
+          <section class="desktop">
+            <div class="desktopIntro">
+              <h2 class="deskHeading"></h2>
+              <!-- <ul class="deskButtons">
+                <li class="deskList1"></li>
+                <li class="deskList2"></li>
+                <li class="deskList3"></li>
+              </ul> -->
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly remove this commented code to improve readability and make your code more professional 💯

**Review by jasonfelice — APPROVED** [review]
Hi @Kasaija-Kenneth, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<p align="center"></p>

## Highlights
No Linter errors✔️
GitHub flow followed✔️
Great design✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -48,127 +47,92 @@ <h1>I'm Kenneth Glad to see you!</h1>
...
-          <div class="hidden2"><button type="button">See Project</button></div>
+      <section id="works">
+        <div id="portfolio" class="works"></div>
+
+        <!-- desktop popup starts from Here -->
+
+        <div id="sectionWrap" class="modal">
+          <section class="desktop">
+            <div class="desktopIntro">
+              <h2 class="deskHeading"></h2>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly try to make the design look as much as possible like it is in Figma. the placement of the modal elements is different, and the size of the image is different too.

## Your design:
image

## Figma design:
image

File: index.html [hunk 3]
```diff
@@ -48,127 +47,92 @@ <h1>I'm Kenneth Glad to see you!</h1>
...
-          </div>
+        <!-----desktop popup ends here---------->
+
+        <!-- mobile popup starts from here -->
+
+        <div id="mobileWrap" class="modal">
+          <section class="mobileDetail">
+            <div class="mobileIntro">
+              <div class="top-Modal">
+                <div>
```
Comment (ha-manel) [inline index.html]: - [ ]  Kindly try to make the mobile design look as much as possible like it is in Figma. the placement of the modal elements is different here too and the technologies badges look very different from the ones in Figma ✍️ 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️ 
* Project cards are created dynamically ✔️

## Required Changes:
_Please check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mateo951 — APPROVED** [review]
### Hi Team, 

Well done making the requested changes by the previous reviewer ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**